rcldotnet_common/DllLoadUtils.cs

```diff
@@ -48,8 +48,8 @@ public class DllLoadUtilsFactory {
       [DllImport ("api-ms-win-core-libraryloader-l1-2-0.dll", EntryPoint = "FreeLibrary", SetLastError = true, ExactSpelling = true)]
       private static extern int FreeLibraryUWP (IntPtr handle);
 
-      [DllImport ("kernel32.dll", EntryPoint = "LoadLibrary", SetLastError = true, ExactSpelling = true)]
-      private static extern IntPtr LoadLibrary (string fileName, int reserved = 0);
+      [DllImport("kernel32", SetLastError = true)]
+      private static extern IntPtr LoadLibrary(string fileName);
```

Comment:
I'd rather keep it with `ExactSpelling`, could you change it to `LoadLibraryW`?

Reply:
`LoadLibraryW` is not working for. I changed to `LoadLibraryA` with `ExactSpelling = true`

Reply:
`LoadLibraryA` is fine as well, it's weird that the unicode version is not available in `kernel32.dll`. Thanks!